pkg/shell/plot.js

```diff
@@ -160,16 +165,33 @@ shell.plot = function plot(element, x_range) {
         walk_timer = null;
     }
 
-    function reset() {
+    function reset(x_range_seconds, x_stop_seconds) {
         stop_walking();
         for (var i = 0; i < data.length; i++)
             data[i].stop();
 
+        // XXX - breaks with DST change during x_range.
```

Comment:
This is how flot wants to be used, I think.  But I agree, we should do our own tick label formatting.

For example, flot doesn't show the day when you are zoomed in enough, so when you scroll far enough into the past, you can't tell what day you are looking at.